t/misc/patch.t

```diff
@@ -184,3 +184,44 @@ m
 
 --- no_error_log
 [error]
+
+
+
+=== TEST 6: resolve host by ourselves (UDP)
+--- yaml_config
+apisix:
+  node_listen: 1984
+  enable_resolv_search_opt: true
+--- config
+    location /t {
+        content_by_lua_block {
+            local sock = ngx.socket.udp()
+            local res, err = sock:setpeername("apisix", 80)
+            if not res then
+                ngx.log(ngx.ERR, err)
+            end
+        }
+    }
+
```

Comment:
omitting the 
```
--- no_error_log
[error]
```
 here, will the test case fail if there is an error level log in error.log?

Reply:
Already added globally: https://github.com/apache/apisix/blob/75376e3162efa3d38ee113a8ee5e6d7435e7c392/t/misc/patch.t#L33